### PR TITLE
issue 434 - $window support in timeout-service

### PR DESCRIPTION
**Pull request by swar30**
implementation for #434

### CODE REVIEW

**Comment by swar30** [comment]
Checking reduced coverage

**Comment by swar30** [comment]
BTW, I'm rather new to writing eslint rules, so would appreciate detailed code review

**Comment by swar30** [comment]
May have found an issue with implementation when running on my own code base, investigating

**Comment by swar30** [comment]
Fixed issues, ready for review

File: docs/timeout-service.md [hunk 1]
```diff
@@ -2,39 +2,61 @@
 
 # timeout-service - use `$timeout` instead of `setTimeout`
 
+
```
Comment (Gillespie59) [inline docs/timeout-service.md]: Why do you have this extra empty lines ?

Comment (swar30) [inline docs/timeout-service.md]: I'm on windows, I think it's CRLF issue, will try to fix

Comment (swar30) [inline docs/timeout-service.md]: @Gillespie59 fixed the extra lines issue

**Comment by swar30** [comment]
@Gillespie59 thanks

**Pull request merged**